proto/booking/pricing/untaxed_reservation_quote.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package holms.types.booking.pricing;
+option csharp_namespace = "HOLMS.Types.Booking.Pricing";
+option go_package = "booking/pricing";
+
+import "booking/pricing/untaxed_reservation_nightly_quote.proto";
+import "booking/pricing/untaxed_incidental_reservation_nightly_quote.proto";
+import "booking/indicators/reservation_indicator.proto";
+
+message UntaxedReservationQuote {
+	bool reservation_subject_to_tax = 1;
```

Comment:
I'd recommend making this lower-level and more granular, e.g. "tax_exemption_requested" or "tax_exemption_applied". Move most of the "thinking" around this into folio, but let booking designate it. 

Also, how are you handling the long-stay TOT exemption? Recall that stays longer than 30 days are deemed exempt from transient occupancy tax. We need to pass the stay length into folio either implicitly, by letting folio count the number of quoted days, or explicitly, by passing a count. What's the plan here?

Reply:
The plan is to count the number of folio days.

I considered passing in a reservation summary. The tricky thing is that estimation is pre-genesis, so the reservation itself doesn't exist yet, which feels wrong to generate a summary object for.

I'm considering making a constructor that throws a special exception if the nightly quotes aren't a contiguous date range.